runtime/indent/idris2.vim

```diff
@@ -0,0 +1,154 @@
+" Vim indent file
+" Language:            Idris2
+" Maintainer:          Serhii Khoma <srghma@gmail.com>
+" Author:              raichoo <raichoo@googlemail.com>
+" Last Change:         2024 Nov 05
+" License:             Vim (see :h license)
+" Repository:          https://github.com/ShinKage/idris2-nvim
+"
+" indentation for idris (idris-lang.org)
+"
+" Based on haskell indentation by motemen <motemen@gmail.com>
+"
+"
+"
+" Modify g:idris_indent_if and g:idris_indent_case to
+" change indentation for `if'(default 3) and `case'(default 5).
+" Example (in .vimrc):
+" > let g:idris_indent_if = 2
+
+if exists('b:did_indent')
+  finish
+endif
+
+let b:did_indent = 1
```

Comment:
Move the 'setl indentexpr' and `setl indentkeys` here to the top and then add some lines like this:
```
  if exists("*GetIdrisIndent")
    finish
  endif
```

Reply:
hope correct